packages/tasit-sdk/.gitignore

```diff
@@ -0,0 +1 @@
+src/config/default.js
```

Comment:
Same question about sharing this file between us outside of GitHub (feel free to resolve this and respond only in the other thread). Just noting that this is relevant in a couple spots.